.github/workflows/pr_e2e_tests.yml

```diff
@@ -0,0 +1,51 @@
+name: Pull request E2E tests
+
+on:
+  pull_request:
+    branches:
+      - V2
```

Comment:
feels like a regression to the previous iteration where you also ran upon `push`

Reply:
The previous iteration did not exist, I never wrote a previous iteration
That was auto generated by playwright
I later removed it to try and fix the confusion